src/device_info.sh

```diff
@@ -0,0 +1,186 @@
+# This file deals with functions related to hardware information
+
+include "$KW_LIB_DIR/remote.sh"
+include "$KW_LIB_DIR/kwlib.sh"
+
+declare -gA shared_data=(['ram']='' ['cpu_model']='' ['cpu_currently']=''
```

Comment:
We will want to use the functions provided here in the deploy and maybe other features, for this reason, rename this to something like `device_info_data`.